src/pages/android-development/index.md

```diff
@@ -17,3 +17,7 @@ Lastly, learn to integrate 3rd party libraries and Firebase services to add func
 ### Official Documentation
 
 [Google Developers Guide for Android](https://developer.android.com/training/index.html)
+
+### Courses
+
+[Udacity Android Nanodegree Program](https://udacity.com/course/android-developer-nanodegree-by-google--nd801
```

Comment:
Hi @AnirudhMukherjee 

Can you please fix the formatting of this link? You forgot a closing parenthesis `)`